tests/main/interfaces-physical-memory-observe/task.yaml

```diff
@@ -0,0 +1,46 @@
+summary: Ensure that the physical memory observe interface works.
+
+details: |
+    The physical-memory-observe interface allows to read the physical memory.
+
+    The test-snapd-physical-memory-observe snap checks that /dev/mem can be read
+    and the interface can be connected and disconnected. 
+
+# As the ubuntu kernels are configured with CONFIG_STRICT_DEVMEM=y, this interface is
+# validated just in the other supported systems.
+systems: [-ubuntu-*]
```

Comment:
I wonder if we could do runtime detection and still let this run on core systems. There it will depend on the kernel and AFAIK will let us validate this for the purpose for which it was added (memory-mapped GPIO toggling via /dev/mem)

Reply:
Can we run on all systems now that you added runtime detection?